tutorials/appengine-dropwizard/index.md

```diff
@@ -0,0 +1,224 @@
+Deploying Dropwizard applications on Google App Engine
+ Flexible Environment is not straightforward as it needs passing arguments 
+ to the JAR, like `java -jar app.jar server config.yaml`.
+ 
+This tutorial shows how to make use of Custom runtime to deploy 
+Apache Dropwizard application using Docker. This
+tutorial assumes that you are familiar with Apache Dropwizard and that you
+have installed Docker.
+
+If you don't have a Dropwizard application already, you can check out
+ [Getting Started Guide][getting-started] to create a sample 
+ Dropwizard application.
+
+[dropwizard]: http://www.dropwizard.io
+[getting-started]: http://www.dropwizard.io/1.2.0/docs/getting-started.html
+[flexible-custom-runtimes]: https://cloud.google.com/appengine/docs/flexible/custom-runtimes/
+
+## Objectives
+
+1. Create a Dockerfile that bundles JAR file and config.yml
+1. Run the Dropwizard app locally with Docker.
+1. Deploy the Dropwizard app to Google App Engine flexible environment.
+
+## Costs
+
+This tutorial uses billable components of Google Cloud Platform, including:
+
+- Google App Engine flexible environment
+
+Use the [Pricing Calculator][pricing] to generate a cost estimate based on your
+projected usage.
+
+[pricing]: https://cloud.google.com/products/calculator
+
+## Before you begin
+
+1.  Create a project in the [Google Cloud Platform Console](https://console.cloud.google.com/).
+1.  Enable billing for your project.
+1.  Install the [Google Cloud SDK](https://cloud.google.com/sdk/).
+
+## Preparing the app
+
+1.  Build your application and generate your JAR artefacts.
+1.  If you don't have a Fat JAR( a JAR file with all the dependencies bundled together ), 
+generate one using a Fat JAR plugin.
+
+    1.  You can use [Apache Maven Shade Plugin][shade]:
+
+        ```
+        <project>
+          ...
+          <build>
+            <plugins>
+              <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-shade-plugin</artifactId>
+                <version>3.1.0</version>
+                <configuration>
+                  <!-- put your configurations here -->
+                </configuration>
+                <executions>
+                  <execution>
+                    <phase>package</phase>
+                    <goals>
+                      <goal>shade</goal>
+                    </goals>
+                  </execution>
+                </executions>
+              </plugin>
+            </plugins>
+          </build>
+          ...
+        </project>
+        ```
+
+    1. Then you can build your app using: 
+    
+        ```
+        mvn package
+        ```
+        
+       This will create a JAR file with all the dependencies of your 
+       application bundled together as one fat file.
+       
+       **Note**: Apache Shader renames your original package JAR 
+       `sample-app-1.0-SNAPSHOT.jar` into 
+       `original-sample-app-1.0-SNAPSHOT.jar` and create the fat jar
```

Comment:
Please be consistent in your use of "fat jar". A quick web search suggests that "fat JAR" is the prevalent capitalization, so I'd suggest using it throughout this doc.